samples/Microsoft.AspNet.Mvc.TagHelpers.Localization/project.json

```diff
@@ -0,0 +1,38 @@
+﻿{
+  "description": "Tag Helpers to aid with localizing ASP.NET MVC applications.",
+  "version": "0.1.0-pre-*",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/aspnet/Entropy"
+  },
+  "compilationOptions": {
+    "warningsAsErrors": true
+  },
+  "dependencies": {
+    "Microsoft.AspNet.Mvc.Localization": "6.0.0-*",
+    "Microsoft.AspNet.Mvc.Razor": "6.0.0-*",
+    "Microsoft.Extensions.Caching.Memory": "1.0.0-*",
```

Comment:
Shouldn't need a bunch of these dependencies. My guess is all you need is `Microsoft.AspNet.Mvc.Razor` and `Microsoft.AspNet.Mvc.Localization`